conf/base.config

```diff
@@ -21,6 +21,13 @@ process {
   maxRetries = 1
   maxErrors = '-1'
 
+  withName:preprocess_bam {
+  cpus = { check_max( 20, 'cpus' ) }
+  memory = { check_max( bam.size() * 5.GB  * task.attempt, 'memory'}
```

Comment:
I think you're missing a closing bracket, so it should be:
`memory = { check_max( bam.size() * 5.GB  * task.attempt, 'memory')}`

Reply:
Yup, sorry :-/

Reply:
No problem